spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe "/ingredients", type: :feature do
+  describe "As a visitor, when I visit the ingredients index page" do
+    before(:each) do
+      @ingredient_1 = Ingredient.create!(name: "Ground Beef", cost: 2)
+      @ingredient_2 = Ingredient.create!(name: "Salt", cost: 4)
+      @ingredient_3 = Ingredient.create!(name: "Butter", cost: 5)
+
+      visit "/ingredients"
+    end
+
+    it "will list of all the ingredients including their name and cost" do
+      expect(page).to have_content(@ingredient_1.name)
```

Comment:
These tests are ok, but since our view has "Name: " as part of the view that displays this data, we shoudl test for it here. 
```
expect(page).to have_content("Name: "#{ingredient_1.name}")
```

That way, when we do the same thing for Cost, if there were 2 ingredients with the same cost, we're more specific about what "2" is. 
Alternatively (or in addition), you could use a `within `block to be even more specific.